third_party/wpt/resources/testharnessreport.js

```diff
@@ -35,6 +35,10 @@ function dump_test_results(tests, status) {
         : document.documentElement;     // fallback for optional <body> in HTML5, SVG, etc.
 
     parent.appendChild(results_element);
+    if (window.testRunner) {
```

Comment:
Not sure we still want to have these fixes in the original file. So the whole purpose of this project is we can have wpt code base unmodified and keep all Cobalt change in a singe layer. I'm pretty sure there will be many other code change like this in testharness.js as well, and we might need to have a better strategy to handle this. And it will be even better if we can write down the few potential solution in the design doc.